samples/dashboard/demos/climate/demo.js

```diff
@@ -38,9 +38,11 @@ const highchartsLogo = [
     'Pf3W9nbKCeFYJFHRcXtorEQkE8KyfD1oKi75LNNPykEm1+rbxeN7/STQvJ/XyS2auivmTPBeh',
     'v5pDgbN4PzsGbOwsI5SIozsk3jH0smSMM9CgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
     'AAAAAAAAAAADg3PAfRDWjQM3VeT0AAAAASUVORK5CYII='
-];
+].join('');
```

Comment:
We are using this logo in multiple places. Wouldn't it be nice to store it somewhere instead of writing this ugly string over and over again?

Reply:
It should be a placeholder in this demo.